cmdx.py

```diff
@@ -1931,13 +1931,18 @@ def clear(self):
         """Remove all members from set"""
         self._fn.clear()
 
-    def sort(self, key=lambda o: (o.typeName, o.path())):
+    def sort(self, key=None):
         """Sort members of set by `key`
 
         Arguments:
             key (lambda): See built-in `sorted(key)` for reference
 
         """
+        if key is None:
+            key = lambda o: (
+                o.typeName if isinstance(o, Node) else None,
+                o.path()
+            )
 
```

Comment:
Could save on indentation levels and remove the if-statement, by:

```py
            key = key or lambda o: (
                o.typeName if isinstance(o, Node) else None,
                o.path()
            )
```

Reply:
oh right! I used to be careful with the or operator. I had a few problems when comparing stuff with None and the result was 0 instead. But there is no chance this will be a problem here